src/ewilgs/models.py

```diff
@@ -1,3 +1,17 @@
-# from django.db import models
+from django.db import models
+from members.models import Members
 
-# Create your models here.
+class Downlink(models.Model):
+    received_at = models.TimeField(auto_now_add=True, null=False)
+    data = models.BinaryField(null=False)
+    frequency = models.FloatField(null=False)
+    processed = models.BooleanField(null=True) #NULL values are allowed for this field
+
+class Uplink(models.Model):
+    UUID_user = models.ForeignKey(Members, editable=False,on_delete=models.CASCADE)
+    created_at = models.TimeField(auto_now_add=True, null=False)
+    transmitted_at = models.TimeField(auto_now=True, null=True) #auto_mow: last edited timestamp is registered
```

Comment:
auto_now in comment 🐱

Reply:
I've been thinking about the auto options for time fields and I think we should have `auto_now=False`, `auto_now_add=False`. This is because the recorded time should never change, and it might be better to add it by ourselves in the code, rather than let the database do it or change it unnecessarily. I believe that way would be safer.

Reply:
I agree. Maybe it is paranoia but I am used to recording reception time in the socket server thread as the first thing, then use that time everywhere afterward. The auto time might bring small differences (usually ms) but it makes frames correlation sometimes tricky.